metricbeat/module/elasticsearch/node_stats/_meta/fields.yml

```diff
@@ -32,127 +30,104 @@
             Total size of the store in bytes.
     - name: jvm.mem.pools
       type: group
-      description: >
-        JVM memory pool stats
```

Comment:
Should these descriptions be removed? Does the doc generator break on group field descriptions?

Reply:
I reviewed this with @dedemorton 
The problem is that each field that is **not leaf** and **has a description** is formatted with "Heading 1". So the only way to maintain coherence with the rest of the fields in the module is to remove the descriptions.
So the easy and fast solution is to remove description in not leaf nodes and to take a look to the fields generation script to see if it can be improved

Reply:
I see, thanks for clarifying. ++ to make these changes here as the immediate solution and then work on making the fields generation script better.